packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -39,6 +38,8 @@ export const {
 // On WWW, __EXPERIMENTAL__ is used for a new modern build.
 // It's not used anywhere in production yet.
 
+export const enableStrictEffects =
+  __DEV__ && dynamicFeatureFlags.enableStrictEffects;
```

Comment:
In theory these `__DEV__` flags shouldn't be needed since every usage of this flag should also be wrapped in `__DEV__`.

Reply:
Yes, they are wrapped.

This change is a follow up to this thread/suggestion:
https://github.com/facebook/react/pull/21590#discussion_r642545020